### PR TITLE
Added russian_peasant_multiplication.c

### PR DESCRIPTION
**Pull request by YashovardhanSiramdas**
**Changes:**
Added Russian Peasant Multiplication in C.

### CODE REVIEW

File: code/mathematical_algorithms/src/russian_peasant_multiplication/russian_peasant_multiplication.c [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#include <stdio.h>
+
+unsigned int 
+russianPeasant(unsigned int a, unsigned int b)
+{
+    int res = 0; 
+
+    while (b > 0) {
```
Comment (arnavb) [inline code/mathematical_algorithms/src/russian_peasant_multiplication/russian_peasant_multiplication.c]: Fix indentation inside this block.

File: code/mathematical_algorithms/src/russian_peasant_multiplication/russian_peasant_multiplication.c [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+         if (b & 1)
+             res = res + a;
```
Comment (arnavb) [inline code/mathematical_algorithms/src/russian_peasant_multiplication/russian_peasant_multiplication.c]: res += a

File: code/mathematical_algorithms/src/russian_peasant_multiplication/russian_peasant_multiplication.c [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        b = b >> 1;
+     }
+
+     return (res);
+}
+
+int 
+main()
+{
+    printf("%u \n", russianPeasant(18, 1));
```
Comment (arnavb) [inline code/mathematical_algorithms/src/russian_peasant_multiplication/russian_peasant_multiplication.c]: No need for the space before the \n. (Nitpicking here)

**Comment by YashovardhanSiramdas** [comment]
Made suggested changes.

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**